### PR TITLE
Add empty state in main workspace area

### PR DESCRIPTION
**Pull request by hojberg**
## Overview
Adds an empty state for when there are no open definitions

Closes https://github.com/unisonweb/codebase-ui/issues/100

Screen Shot 2021-07-25 at 20 33 52

### CODE REVIEW

File: src/App.elm [hunk 1]
```diff
@@ -336,10 +336,10 @@ viewMainSidebar model =
...
         [ div [] [ Html.map CodebaseTreeMsg (CodebaseTree.view model.codebaseTree) ]
         , nav []
-            [ a [ href "https://unison-lang.org", title "Unison website", rel "noopener", target "_blank" ] [ Icon.view Icon.unisonMark ]
-            , a [ href "https://unison-lang.org/docs", rel "noopener", target "_blank" ] [ text "Docs" ]
-            , a [ href "https://unison-lang.org/docs/language-reference", rel "noopener", target "_blank" ] [ text "Language Reference" ]
-            , a [ href "https://unison-lang.org/community", rel "noopener", target "_blank" ] [ text "Community" ]
+            [ a [ href "https://unisonweb.org", title "Unison website", rel "noopener", target "_blank" ] [ Icon.view Icon.unisonMark ]
+            , a [ href "https://unisonweb.org/docs", rel "noopener", target "_blank" ] [ text "Docs" ]
+            , a [ href "https://unisonweb.org/docs/language-reference", rel "noopener", target "_blank" ] [ text "Language Reference" ]
+            , a [ href "https://unisonweb.org/community", rel "noopener", target "_blank" ] [ text "Community" ]
```
Comment (hojberg) [inline src/App.elm:342]: This also fixes https://github.com/unisonweb/codebase-ui/issues/166

File: src/UI/Button.elm [hunk 2]
```diff
@@ -229,7 +229,7 @@ sizeToClassName size =
             "medium"
 
         Large ->
-            "secondary"
+            "large"
```
Comment (hojberg) [inline src/UI/Button.elm:232]: Bug with class names for button sizes.

**Review by pchiusano — APPROVED** [review]
Great! Minor thing - "Checkout" should be "Check out". https://grammarist.com/spelling/checkout-check-out/

**Comment by hojberg** [comment]
@pchiusano hah, I guess my payments background is showing :D Good catch

**Pull request merged**